pkg/report/linux.go

```diff
@@ -768,6 +768,7 @@ func warningStackFmt(skip ...string) *stackFmt {
 			parseStackTrace,
 		},
 		parts2: []*regexp.Regexp{
+			compile("WARNING: .* {{FUNC}}"),
```

Comment:
Is this needed? What will change if this is removed?
As the comment above states, this is for old warning format. Your new report should be parsed using parts, not parts2. But even for old reports, shouldn't the {{FUNC}} be present in the Call Trace too?

Reply:
I've added this to extract meaningful titles from corrupted reports (e.g. [25](https://github.com/google/syzkaller/blob/master/pkg/report/testdata/linux/report/25) or [342](https://github.com/google/syzkaller/blob/master/pkg/report/testdata/linux/report/342)), which don't have a stack trace. Although now I'm thinking that this won't use `skipPatterns` for the old warning format (since `%[3]v` will match to `{{FUNC}}` in `parts2`). Maybe we need something like `parts3` for corrupted reports...

Reply:
We don't need a meaningful title for corrupted reports. It's discarded anyway, we can't trust it.

Reply:
OK, PTAL. I've had to mark some arm crash logs as corrupted though.

Reply:
The new corrupted reports look indeed corrupted, they don't contain meaningful stack traces. Some are attributed to frames that should be skipped. So I think this is the right thing to do.